README.md

```diff
@@ -27,11 +27,67 @@ security of the system, such as auditing and explainability features.
 
 [SIG-Security charter](governance/charter.md) outlines the scope  of our group activities, as part of our [governance process](governance) which details how we work.
 
-## Members
+## Communications
+
+Anyone is welcome to join our open discussions of SIG-Security projects and share news related to the group's mission and charter. Much of the work of the group happens outside of SIG-Security meetings and we encourage project teams to share progress updates or post questions in these channels:
+
+* [Email list](https://lists.cncf.io/g/cncf-sig-security)
+* [CNCF Slack](https://slack.cncf.io/) #sig-security channel
+
+## Meeting Time
+
+The Security SIG group meets every Wednesday at 10:00am PT (USA Pacific):
+
+[Meeting minutes and agenda](https://docs.google.com/document/d/170y5biX9k95hYRwprITprG6Mc9xD5glVn-4mB2Jmi2g/)
+[Meeting Link: zoom.us/my/cncfsigsecurity](https://zoom.us/my/cncfsigsecurity)
```

Comment:
for some reason it's putting both of these on one line - maybe MD is treating this as one paragraph? it'd probably be fixed if you used bullets here.

Reply:
Good catch! Fixed!

Reply:
looks good on github preview!